### DISCUSSION TITLE
Hyperlink buttons in address column that redirect to meeting detail page.

### DISCUSSION THREAD

**Discussion by toddjsaunders** [code4recovery/12-step-meeting-list#666] [Q&A] [answered]
AA SF/Marin has Zoom and Phone buttons in their address column that take the user to the meeting detail page, I find this more intuitive than the user having to click the meeting name.. and it gives people two ways to get to the detail page.

(I also see that their Online meetings don't have default "Online" in the location column)

I'd like to replicate this on our site, any help would be appreciated.

Thanks!

https://aasfmarin.org/find-a-meeting

<img width="577" alt="Screen Shot 2022-02-03 at 2 14 28 PM" src="https://user-images.githubusercontent.com/8978087/152438539-493cd125-2b43-4c03-b6a7-b06f2f09a113.png">

**Comment by kiyote33** [comment, marked as answer]
You can embed this shortcode in the meetings page: **[tsml_react]**. This is a new feature being used on select sites which we are getting very positive feedback on. It replaces the legacy front end code.  Let us know if you need more info...
You can also use **[tsml_ui]** which I believe is the preferred shortcode.

**Reply by toddjsaunders** [reply]
Thanks so much!  The new UI gets my vote :)

**Reply by toddjsaunders** [reply]
Hi again,

As I'm configuring this, two questions came up --

With the new front end, our online meetings say both "Online" and "Online Meeting" but on the old front end they only say "Online"

![Screen Shot 2022-02-03 at 6 06 41 PM](https://user-images.githubusercontent.com/8978087/152460862-2fb3c32a-27c4-4e26-96ef-cb21ee5b1a1a.png)

![Screen Shot 2022-02-03 at 6 06 30 PM](https://user-images.githubusercontent.com/8978087/152460876-df179c10-fa27-4957-8755-b492dc4188c6.png)

I'd like to remove "Online Meeting" from showing up...

Also, on the in-person meetings is there a way to not show the list of other meetings at the same location?

![Screen Shot 2022-02-03 at 6 06 51 PM](https://user-images.githubusercontent.com/8978087/152461056-7a4a9af7-c707-4744-bc78-6153dc2883e6.png)

Thanks!  Let me know if this would be better in a new discussion, rather than added on here :)

**Reply by kiyote33** [reply]
Definitely should be a new discussion.  We should get the TSML UI team involved on your questions. The 2 Online types don't show up in the legacy 12 Step Meeting List when reviewing online meetings. As for a method to hide the **Additional Meetings At This Location** list, that too would be a question for the TSML UI team.  The old legacy TSML simply notes the number of additional meetings at the location without listing them all out.
